### DISCUSSION TITLE
Welcome to WhereDoggo Discussions!

### DISCUSSION THREAD

**Discussion by IntegerMan** [IntegerMan/WhereDoggo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hdugzx** [comment]
Hello, I major in AI and it's my first year to study AI in HDU. When I open this project I saw the technology of AI before the Chatgpt came out, and I really interested in that, btw, now I'm trying to achieve AI-werewolves. I'll try my best to make this project better!
